src/utilities/effects.ts

```diff
@@ -511,9 +511,11 @@ const effects = {
     }
   },
 
-  async createSchedulingSpec(spec: SchedulingSpecInsertInput): Promise<void> {
+  async createSchedulingSpec(spec: SchedulingSpecInsertInput): Promise<Pick<SchedulingSpec, 'id'>> {
```

Comment:
Is the type here correct? Are you expecting to return a full scheduling spec including and id? This would just give the id I believe.

Reply:
Ah right I think it is only returning an id. Should I just type that explicitly as `{ id: number }`, or should there be an actual named type for this?

Reply:
Ah sorry you are good! I thought the plan included the entire spec and not just and id. Nothing to see here.